// string-checker/bin/yargs.config.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
const yargs_1 = __importDefault(require("yargs"));
exports.argv = yargs_1.default
    .command('string-checker', 'Checks for given strings in specified directory')
    .option('dir', {
    alias: 'd',
    demandOption: true,
    description: 'specifies the root directory for file search',
    type: 'string',
})
    .option('searchString', {
    alias: 's',
    array: true,
    demandOption: true,
    description: 'text to search for',
    type: 'string',
})
    .option('exclusions', {
    alias: 'x',
    array: true,
    description: 'files that are excluded from the search',
    type: 'string',
})
    .option('error', {
    alias: 'e',
    default: false,
    description: 'returns an error rather than a warning',
    type: 'boolean',
})
    .option('console', {
    alias: 'c',
    default: true,
    description: 'outputs results to console',
    type: 'boolean'
})
    .help()
    .alias('help', 'h')
    .argv;
